changelog.md

```diff
@@ -16,6 +16,7 @@ Bug Fixes:
 ----------
 
 * Fix an error when using login paths with an explicit database name (Thanks: [Thomas Roten]).
+* The \o command could only be used once per session (Thanks: [Georgy Frolov])
```

Comment:
This PR is doing the opposite of allowing \o only once in a session, right? 

Shouldn't this say: "Allow \o command more than once per session".

Reply:
Agree, clumsy wording. I meant to say "Fixed a bug where it could only be used...".
Besides, this line was in the wrong section :)